### DISCUSSION TITLE
Is it possible to import third-party react native libraries?

### DISCUSSION THREAD

**Discussion by jimzqw** [necolas/react-native-web#2449] [Q&A] [answered]
### Is there an existing request?

- [X] I have searched for this request

### Describe the feature request

Might be naive but is it possible to import third-party RN libraries? For example: react-native-month-year-picker

**Comment by Dovakeidy** [comment, marked as answer]
You can use all the RN libraries you want for the native part. However, not all of them are web compatible. I advise you to look at this site to know which ones are : https://reactnative.directory/?web=true
